app/views/brexit_campaign/_featured_taxons.html.erb

```diff
@@ -1,18 +1,15 @@
-<section>
-  <% @taxons.in_groups_of(3, false).each_with_index do |taxons_group, row_index| %>
-    <div class="grid-row">
-      <% taxons_group.each do |taxon| %>
-        <div class="column-one-third">
-          <div class="campaign__taxon <%= "campaign__taxon--variable-border" if row_index.eql?(0) %> ">
-            <h3 class="campaign__taxon-title">
-              <a href="<%= taxon.finder_link %>"
-                 aria-label="<%= t('campaign.see_more_in_taxon', topic: taxon.title.downcase) %>"><%= taxon.title %></a>
-              </h3>
-
-            <p class="campaign__description"><%= taxon.description %></p>
-          </div>
+<% @taxons.in_groups_of(local_assigns.fetch(:display_full_width, true) ? 3 : 2, false).each_with_index do |taxons_group, row_index| %>
```

Comment:
If we need to keep the logic associated with `display_full_width` (see my other comment), we may want to pull this check (and the one on line 4) into variables at the top of the page to help make the HTML a bit more readable.

Reply:
Noted.